### ISSUE TITLE
Vehicle accessibility_attributes type inconsistency

### ISSUE THREAD

**Issue by johnzuk** [CityOfLosAngeles/mobility-data-specification#947]
**Describe the bug**
Vehicle `accessibility_attributes` is inconsistent between different modes

**To Reproduce**
accessibility_attributes is Enum[] in all modes except delivery-robots where it is an object with bool only attributes

**Expected behavior**
accessibility_attributes should be consistent between all modes as a Enum[]

Instead of:

The `accessibility_attributes` object **may** have the following key value pairs:

- `audio_cue` (boolean, [Optional](../general-information.md#optional-fields)): is the device equipped with audio cues upon delivery
- `visual_cue` (boolean, [Optional](../general-information.md#optional-fields)): is the device equipped with visual cues upon delivery
- `remote_open` (boolean, [Optional](../general-information.md#optional-fields)): can the device door be remotely opened to retrieve cargo upon delivery

It should be:

### Accessibility Attributes

This `accessibility_attributes` enum represents the accessibility attributes available on a given vehicle, or the accessibility attributes utilized for a given trip. 

| `accessibility_attributes` | Description                           |
|----------------------------|---------------------------------------|
| `audio_cue`    | is the device equipped with audio cues upon delivery |
| `visual_cue`    | is the device equipped with visual cues upon delivery |
| `remote_open`    | can the device door be remotely opened to retrieve cargo upon delivery |

**Comment by schnuerle** [comment]
Agree thanks for seeing this. We can work to fix it in MDS 2.1.

**Comment by schnuerle** [comment]
Fixed with this [commit](https://github.com/openmobilityfoundation/mobility-data-specification/pull/965/commits/324c1672e765f59c3a940116f9f2099926465894).
